aten/src/ATen/NestedTensorImpl.cpp

```diff
@@ -179,7 +179,7 @@ NestedTensorImpl::NestedTensorImpl(
       "in the near future.");
   auto storage_device = storage_.device();
   TORCH_INTERNAL_ASSERT(
-      storage_device.is_cpu() || storage_device.is_cuda() || storage_device.is_privateuseone(),
+      storage_device.is_cpu() || storage_device.is_cuda() || storage_device.is_xpu() || storage_device.is_privateuseone(),
       "NestedTensorImpl storage must be either CUDA, CPU or ", get_privateuse1_backend(), " but got ",
```

Comment:
shall we update this assertion message as well?

Reply:
OK. Fix it. Thank you for comments. :)